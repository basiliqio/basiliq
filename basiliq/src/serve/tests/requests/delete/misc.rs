use super::*;

crate::run_test_request!(
    sorting_local,
    Method::DELETE,
    format!("/public__peoples/{}?sort=first-name", PEOPLES_IDS[0]),
    200
);

crate::run_test_request!(
    sorting_local_multi,
    Method::DELETE,
    format!(
        "/public__peoples/{}?sort=first-name,-last-name",
        PEOPLES_IDS[0]
    ),
    200
);

crate::run_test_request!(
    sorting_remote,
    Method::DELETE,
    format!(
        "/public__peoples/{}?sort=-public__favorite_color.color",
        PEOPLES_IDS[0]
    ),
    200
);

crate::run_test_request!(
    sorting_remote_multi_rel,
    Method::DELETE,
    format!(
        "/public__peoples/{}?sort=public__articles.title",
        PEOPLES_IDS[0]
    ),
    200
);

crate::run_test_request!(
    sorting_and_include_and_sparse,
    Method::DELETE,
    format!("/public__peoples/{}?sort=public__favorite_color.color&include=public__favorite_color&fields[public__favorite_color]=", PEOPLES_IDS[0]),
    200
);
