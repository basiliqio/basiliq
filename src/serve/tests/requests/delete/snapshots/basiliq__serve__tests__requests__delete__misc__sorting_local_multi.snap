---
source: basiliq/src/serve/tests/requests/delete/misc.rs
expression: res

---
{
  "jsonapi": {
    "version": "1.0"
  },
  "data": null
}
